test/test_w3c_spec/test_trix_w3c.py

```diff
@@ -0,0 +1,129 @@
+"""This runs the nt tests for the W3C RDF Working Group's N-Quads
```

Comment:
I think maybe this should rather go into `test/test_suites/test_trix.py` rather than `test/test_w3c_spec/test_trix_w3c.py` as this is not a w3c suite as far as I understand.

Reply:
Sure.

Reply:
Happy with the new name.